packages/grid/_modules_/grid/components/toolbar/GridToolbarExport.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import MenuList from '@material-ui/core/MenuList';
+import Button from '@material-ui/core/Button';
+import MenuItem from '@material-ui/core/MenuItem';
+import { ApiContext } from '../api-context';
+import { useGridSelector } from '../../hooks/features/core/useGridSelector';
+import { optionsSelector } from '../../hooks/utils/optionsSelector';
+import { GridMenu } from '../menu/GridMenu';
+import { ExportOption } from '../../models';
+
+export function GridToolbarExport() {
+  const apiRef = React.useContext(ApiContext);
+  const options = useGridSelector(apiRef, optionsSelector);
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const ExportIcon = apiRef!.current.components!.ExportIcon!;
+
+  const ExportOptions: Array<ExportOption> = [
+    {
+      label: apiRef!.current.getLocaleText('toolbarExportCSV'),
+      format: 'csv',
+    },
+  ];
+
+  const handleExportSelectorOpen = (event) => setAnchorEl(event.currentTarget);
+  const handleExportSelectorClose = () => setAnchorEl(null);
+  const handleExport = (format) => {
+    if (format === 'csv') {
+      apiRef!.current.exportDataAsCsv();
+    }
+
+    setAnchorEl(null);
+  };
+
+  const handleListKeyDown = (event: React.KeyboardEvent) => {
+    if (event.key === 'Tab' || event.key === 'Escape') {
+      event.preventDefault();
+      handleExportSelectorClose();
+    }
+  };
+
+  // Disable the button if the corresponding is disabled
+  if (options.disableCsvExport) {
+    return null;
+  }
+
+  const renderExportOptions: Array<React.ReactElement> = ExportOptions.map((option, index) => (
+    <MenuItem key={index} onClick={() => handleExport(option.format)}>
+      {option.label}
+    </MenuItem>
+  ));
+
+  return (
+    <React.Fragment>
+      <Button
+        color="primary"
+        size="small"
+        startIcon={<ExportIcon />}
+        onClick={handleExportSelectorOpen}
+        aria-expanded="true"
+        aria-haspopup="listbox"
```

Comment:
My bad, I didn't link the right pattern. Here, I believe that it should be the same as this demo: https://next.material-ui.com/components/menus/#basic-menu or https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html

Reply:
Updated, as far as I saw it needs a value of `true` in that case and no role is needed for the menu.